srcpkgs/hctl/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'hctl'
+pkgname=hctl
+version=0.2.2
+revision=1
+build_style=cargo
+hostmakedepends="pkg-config"
+makedepends="openssl-devel"
+depends="openssl"
```

Comment:
not needed, this should be automatically detected

Reply:
When removed `build_style=cargo` got:
```bash
=> ERROR: hctl-0.2.2_1: missing do_install() function!

```

When removed:
```ini
hostmakedepends="pkg-config"
makedepends="openssl-devel"
depends="openssl"
```
Got message:
`(...) The pkg-config command could not be found. (...)`

And when added pkg-config
`Package openssl was not found in the pkg-config search path.`

Are you sure?

Reply:
I'm talking about depends=openssl

Reply:
Right, it passes without `depends=openssl`

One more thing. Alpine Maintainers [protested against](https://gitlab.alpinelinux.org/alpine/aports/-/merge_requests/51062#note_335395) the [BSD-3-Clause-**No-Military**](https://spdx.org/licenses/BSD-3-Clause-No-Military-License.html) so I must have changed it to just [BSD-3-Clause](https://spdx.org/licenses/BSD-3-Clause.html). I rebased amended and force pushed [code for 0.2.2 commit](https://github.com/michalszmidt/hctl/commit/7897bd115bf279c08b0971509006afc375dd6fae) adding new license there but old archive is glued to release. So i made a new tag 0.2.2.1 with modified license.

So next force push contains also License change and checksum update.
Also `revision=0`as this new package.